front/src/services/api/items.ts

```diff
@@ -51,3 +55,13 @@ export async function deleteItem(
 
   return response.data;
 }
+
+export async function getAdminItem(id: number): Promise<ItemResponse> {
+  const response = await apiClient.get<ItemResponse>(`/admin/items/${id}`);
```

Comment:
shared/typesに定義した型を使っていただきたいです
```suggestion
  const response = await apiClient.get<GetRes['admin/items/:id']>(`/admin/items/${id}`);
```